charts/stardog/README.md

```diff
@@ -53,10 +53,45 @@ Configuration Parameters
 | `busybox.image.tag`                          | The Docker image tag for busybox image (used as a part of Stardog initialization) |
 | `busybox.image.username`                     | The Docker registry username for busybox image registry (used as a part of Stardog initialization) |
 | `busybox.image.password`                     | The Docker registry password for the busybox image registry (used as a part of Stardog initialization)  |
-| `additionalStardogProperties`                | Allow adding additional settings to stardog.properties file |
+| `additionalStardogProperties`                | Allows adding additional settings to stardog.properties file |
 
 The default values are specified in `values.yaml` as well as the required values for the ZooKeeper chart.
 
+Memory Parameters
+-----------------
+
+Memory settings are part of both javaArgs and resources within
+values.yaml.  The settings must be coordinated.
+
+Stardog will aggressively use all memory allocated to it via the
+javaArgs settings of -Xmx and -XX:MaxDirectMemorySize.  The total
+memory used by Stardog is the sum of -Xmx and -XX:MaxDirectMemorySize.
+
+The resources settings of requests:memory and optional limits:memory
+must account for the javaArgs settings.  requests:memory should be at
+least 1g larger than -Xms and -XX:MaxDirectMemorySize total. 2g larger is
+preferred.  If you chose to establish resourse:limits, limits:memory
+must be at least 1g larger than -Xmx and -XX:MaxDirectMemorySize total.  A
+too small limits:memory setting will likely result in a system crash.
+
+Network based disk storage
+--------------------------
+
+Network based disk storage is not recommended.  This include NFS, SMB, and
+EFS based storage systems.
+
```

Comment:
On AWS, we recommend `gp2` as the `storageClass`. On GKS and AKS `standard` (the default) is what we recommend. I would just add something like "we instead recommend setting the `storageClass` to `gp2` on AWS and leaving the default values of `standard` for Azure and GCP."

Reply:
Just to clarify from discussions outside of this PR. I think we should call out a bit more details that the storage classes may not always be named this depending on how they are configured. Instead we should tell people to use block volume storage classes for their infrastructure and depending on their use case they may want to explore differences between the types, e.g. `gp2` vs `io2` or `standard` vs `premium`. We can also link to the k8s docs on storage classes for further information: https://kubernetes.io/docs/concepts/storage/storage-classes/

Reply:
To include the suggestions from Paul's comment, I would edit it to read something like: 
Stardog instead recommends using block volume storage classes such as gp2 on AWS or standard on other platforms. Depending on use case, it is recommended to explore differences between the types, e.g. gp2 vs io2 or standard vs premium.